Dockerfile

```
@@ -120,13 +133,19 @@ RUN set -eux; \
+# Adding new non-root user to pass CI warn/check for reaching caddy as root.RUN addgroup -S caddygroup && adduser -S caddyuser -G caddygroup
+RUN addgroup -S caddygroup && adduser -S caddyuser -G caddygroup
 
 WORKDIR /srv/app
 
 COPY --from=app_caddy_builder --link /usr/bin/caddy /usr/bin/caddy
-COPY --from=app_php --link /srv/app/public public/
-COPY --link infrastructure/docker/caddy/Caddyfile /etc/caddy/Caddyfile
+COPY --chown=caddyuser:caddygroup --link --from=app_php /srv/app/public /srv/app/public
+COPY --chown=caddyuser:caddygroup --link infrastructure/docker/caddy/Caddyfile /etc/caddy/Caddyfile
+RUN chown -R caddyuser:caddygroup /srv/app /etc/caddy \# Встановлюємо власника для всієї структури
+    && chmod -R a-w /srv/app/public
```

Hadolint (2.12.0)

[error] unexpected '&'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)